mod/developers/views/default/theme_sandbox/forms.php

```diff
@@ -103,6 +103,22 @@
 			'label' => 'Date input (.elgg-input-date):',
 		));
 
+		echo elgg_view_input('date', array(
+			'name' => 'f12-custom',
+			'id' => 'f12-custom',
+			'value' => '1-Jan-2016',
+			'timestamp' => true,
+			'datepicker_options' => array(
+				'dateFormat' => 'd-M-yy',
+				'changeMonth' => false,
+				'changeYear' => false,
+				'minDate' => '1-Jan-2016',
```

Comment:
These short month names have to match the language that datepicker uses. So numbers work best. PR coming.